README.md

```diff
@@ -50,3 +50,26 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+A Python Trello API utility to Create, move and delete Cards from To Do, Doing, Done Lists
+Create a Card using board id into to do List 
+Move the card from To Do list to Doing List using Card id
+Move the cards from Doing list to done List using Done List Id
+Delete a Card using Card id
+
+Set Up
+Register with Trello
+Generated api key and token  for the Trello account
+Saved api key and token in env
+
+Create Classes
+Trello
+Items
+app
+
+Run the Tests
+poetry run flask run
+
+Changes to template
+index.html
```

Comment:
This update to your README looks out of place. You can see how it appears if you scroll to the bottom of this page:
https://github.com/Arpitha-debug/DevOps-Course-Starter

Or you can preview it in VSCode. 

Your README should include instructions for getting set up and running the app/tests. So you should keep something like this section:
```
Set Up
Register with Trello
Generated api key and token  for the Trello account
Saved api key and token in env
```

But you should fix the formatting and reword it so it's clearer.